src/content/docs/logs/ui-data/parsing.mdx

```diff
@@ -15,7 +15,7 @@ Parsing is the process of splitting unstructured log data into attributes (key/v
 
 To get started with parsing, you may want to watch the following video tutorial on YouTube (approx. 4-1/2 minutes).
 
-<Video id="PD7zGXqS0P0" type="youtube" width="560px" />
+//<Video id="PD7zGXqS0P0" type="youtube" width="560px" />
```

Comment:
Not sure why these forward slashes were added. Are you trying to comment them out @thezackm ?

Reply:
ugh, nope! I commented those out to get my markdown preview working is VS Code and forgot to remove them 😬 

fixing now!